apps/mobile/src/features/Settings/useCases/changeAppSettings/ApplicationSettingsScreen.tsx

```diff
@@ -22,17 +29,40 @@ import {useCrashReports} from './useCrashReports'
 
 export const ApplicationSettingsScreen = () => {
   const strings = useStrings()
-  const {atoms: ta} = useTheme()
-  const {network} = useSelectedNetwork()
-  const {isPrivacyActive} = usePrivacyMode()
-  const crashReports = useCrashReports()
-  const screenShareQuery = useScreenShareSettingEnabled()
+  const {atoms: ta, paletteName: name, palette: p} = useTheme()
+  const {languageCode} = useLanguage()
+  const language = supportedLanguages.find(
+    (lang) => lang.code === languageCode,
+  ) as LanguageRecord
+
+  const {isTogglePrivacyModeLoading, isPrivacyActive} = usePrivacyMode()
   const {currency} = useCurrencyPairing()
+  const {enabled: crashReportEnabled} = useCrashReports()
+
+  const authSetting = useAuthSetting()
+  const isAuthOsSupported = useIsAuthOsSupported()
   const navigateTo = useNavigateTo()
 
-  const isScreenSharingDisabled = React.useMemo(() => {
-    return Platform.OS === 'ios' && network === 'mainnet'
-  }, [network])
+  const {authWithOs} = useAuthWithOs({onSuccess: navigateTo.enableLoginWithPin})
+
+  const {network} = useSelectedNetwork()
+
+  const {data: screenShareEnabled, isLoading: isScreenShareLoading} =
+    useScreenShareSettingEnabled()
+  const displayScreenShareSetting = Platform.OS === 'android'
```

Comment:
this is static, can be moved outside the component, there is a helper for it though, isAndroid from the env/constants